README.md

```diff
@@ -1 +1,97 @@
-# BumberBros-Api
\ No newline at end of file
+# BumberBros-Api
+
+## Overview
+Bumper is a turn-based strategy game where up to 4 players navigate a grid to collect power-ups and engage in combat. Players input their movement directions and the game executes them in sequence. The goal is to outmaneuver and outfight your opponents using strategic moves and power-ups.
```

Comment:
Just need to change this first word to `Bumber Bros` (or is it `BumberBros`?)

Reply:
Good catch let me fix that